<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>addContact</class>
 <widget class="QDialog" name="addContact">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>250</width>
    <height>191</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add Contact</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>231</width>
     <height>171</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="label_firstname">
          <property name="text">
           <string>First Name</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_lastname">
          <property name="text">
           <string>Last Name</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_terminalid">
          <property name="text">
           <string>Terminal-ID</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_unitname">
          <property name="text">
           <string>Unit Name</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLineEdit" name="lineEdit_firstname"/>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_lastname"/>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_terminalid"/>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_unitname"/>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_save">
      <property name="text">
       <string>Save</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
